From e3acac792662ce3eeadfe98c4ee99476c1aee355 Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Tue, 5 Feb 2019 16:52:41 -0800
Subject: [PATCH] MANTA-4092 Binder loses race with Zookeeper on startup
 leading to detached zk session watchers Reviewed by: Joshua M. Clulow
 <jmc@joyent.com> Approved by: Joshua M. Clulow <jmc@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 61a9dca..943a519 100644
--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "vasync": "2.1.0",
     "verror": "^1.10.0",
     "xtend": "1.0.3",
-    "zkstream": "0.11.0"
+    "zkstream": "0.11.3"
   },
   "engines": {
     "node": ">=0.10"
-- 
2.21.0

